core/classes/Core/Input.php

```diff
@@ -53,10 +53,56 @@ public static function get(string $item) {
      *
      * @param Language $language Instance of language class to use for translation.
      * @param string $name Name of input field ID.
+     * @param bool $mentions Whether to enable mention autocompletion/parsing or not.
      */
-    public static function createTinyEditor(Language $language, string $name): string {
+    public static function createTinyEditor(Language $language, string $name, bool $mentions = false): string {
         $skin = defined('TEMPLATE_TINY_EDITOR_DARKMODE') ? 'oxide-dark' : 'oxide';
-        return "
+        $js = '';
+
+        if ($mentions) {
+            $js .= "
+                (function () {
+                    let flags = (function () {
+                        'use strict';
+                        tinymce.PluginManager.add('mentions', function (editor, url) {
+                            editor.ui.registry.addAutocompleter('autocompleter-mentions', {
+                                ch: '@',
+                                minChars: 2,
+                                columns: 1,
+                                fetch: function (pattern) {
+                                    return new tinymce.util.Promise(function (resolve) {
+                                        fetch('" . URL::build('/queries/mention_users') . "&nickname=' + pattern)
```

Comment:
Not sure if this will work if friendly URLs are enabled as the URL would be
```
<domain>/queries/mention_users&nickname=abc
```
ie there is no `?`

This could work as a "hack":
```php
URL::build('/queries/mention_users', 'nickname') . "='
```